api/api_test.go

```diff
@@ -29,3 +37,94 @@ func testHTTPServer(
 
 	return config, ln
 }
+
+// nextPort is the next port to use for the API server.
+var nextPort int32 = 28200
+
+// restVaultServer runs an instance of the Vault server in development mode.
+// This requires that the vault binary is installed and in the $PATH.
+func testVaultServer(t *testing.T) (*Client, func()) {
```

Comment:
@ryanuber See my next comment about using a `helpers/testing` package. Just put the tests elsewhere and there is no concern with a cyclic dependency.

@bk You could but you can't have two packages in one directory, so it'd have to live elsewhere anyways.